test/pod.bats

```diff
@@ -343,19 +242,10 @@ function teardown() {
 
 	config=$(cat "$TESTDATA"/sandbox_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);del obj["linux"]["cgroup_parent"]; json.dump(obj, sys.stdout)')
 	echo "$config" > "$TESTDIR"/sandbox_config-systemd.json
-	run crictl runp "$TESTDIR"/sandbox_config-systemd.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	pod_id="$output"
-
-	run crictl create "$pod_id" "$TESTDATA"/container_config_sleep.json "$TESTDIR"/sandbox_config-systemd.json
-	echo "$output"
-	[ "$status" -eq 0 ]
-	ctr_id="$output"
+	pod_id=$(crictl runp "$TESTDIR"/sandbox_config-systemd.json)
+	ctr_id=$(crictl create "$pod_id" "$TESTDATA"/container_config_sleep.json "$TESTDIR"/sandbox_config-systemd.json)
 
-	run crictl start "$ctr_id"
-	echo "$output"
-	[ "$status" -eq 0 ]
+	crictl start "$ctr_id"
 
 	run systemctl show "crio-${ctr_id}.scope"
```

Comment:
Here, as well, since 0 exit code is expected, you can do `output=$(systemctl show "crio-${ctr_id}.scope")`. Sorry I missed it earlier.